### PR TITLE
Custom error pages added

### LINKED ISSUE #280: Create a custom 500 and 404 error page in for django
For reference, see this http://stackoverflow.com/questions/17662928/django-creating-a-custom-500-404-error-page

### PR DESCRIPTION
**Pull request by anweshknayak**
fixed #280

### CODE REVIEW

**Comment by anweshknayak** [comment]
@deshraj review please

File: apps/challenges/views.py [hunk 1]
```diff
@@ -181,3 +181,16 @@ def get_all_challenges(request, challenge_time):
...
+                                  context_instance=RequestContext(request))
+    response.status_code = 404
+    return response
+
+
+def handler500(request):
+    response = render_to_response('src/views/web/error/500.html', {},
+                                  context_instance=RequestContext(request))
+    response.status_code = 500
+    return response
```
Comment (deshraj) [inline apps/challenges/views.py]: @anweshknayak you don't need to write this piece of code in all the repositories. You can just create this in evalai/views.py and put a route for the same in evalai/urls.py

**Comment by deshraj** [comment]
Also, can you please put simple html content in 404.html and 500.html and we can design it later.

**Comment by deshraj** [comment]
Also, please make sure that your travis builds are passed.

**Comment by anweshknayak** [comment]
@deshraj take a look please. I don't understand why some checks are failing ?

File: frontend/src/views/web/error/404.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<html>
+<header></header>
+<body>
+
```
Comment (deshraj) [inline frontend/src/views/web/error/404.html]: Can you add some content like `<h1>404 error occured</h1>`?

File: frontend/src/views/web/error/500.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<html>
+<header></header>
+<body>
+
```
Comment (deshraj) [inline frontend/src/views/web/error/500.html]: Can you add some content like `<h1>500 Internal Server Error</h1>`?

File: evalai/views.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+from django.shortcuts import render_to_response
+from django.template import RequestContext
+
+# HTTP Error 404
```
Comment (deshraj) [inline evalai/views.py]: please delete this line and make sure there is a gap of two lines only before starting the function definition?

File: evalai/views.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+
+def page_not_found(request):
+    response = render_to_response('src/views/web/error/404.html',
+                                  context_instance=RequestContext(request)
+                                  )
+    response.status_code = 404
+    return response
+
+# HTTP Error 500
```
Comment (deshraj) [inline evalai/views.py]: Please delete this also.

File: settings/common.py [hunk 6]
```diff
@@ -21,6 +21,12 @@
 
 sys.path.append(APPS_DIR)
 
+TEMPLATE_PATH = os.path.join(BASE_DIR, 'frontend')
+TEMPLATE_DIRS = (
+    # Put strings here, like "/home/html/django_templates" or "C:/www/django/templates".
+    # Don't forget to use absolute paths, not relative paths.
```
Comment (deshraj) [inline settings/common.py]: Can you remove these lines 25 and 26?

**Comment by deshraj** [comment]
@anweshknayak see this https://travis-ci.org/Cloud-CV/EvalAI/builds/186551116. 

See the flake8 errors shown in the log. Let me know if you cannot find this. 
The lines would be like this: 

```
$ flake8 ./
./evalai/urls.py:27:1: F811 redefinition of unused 'handler404' from line 17
./evalai/urls.py:28:1: F811 redefinition of unused 'handler500' from line 17
```

**Comment by anweshknayak** [comment]
@deshraj yes I tried to fix those flake error before too. I am not getting the idea where it is stuck.

**Comment by deshraj** [comment]
@anweshknayak As you can see that the coverage is decreased, so can you please add test cases for both of these views. Please let me know if you need help with this.

**Pull request closed without merging**